.vscode/launch.json

```diff
@@ -94,6 +98,26 @@
             "sourceMaps": true,
             "outDir": "${workspaceRoot}/out",
             "preLaunchTask": "build"
+        },
+        {
+            "name": "Run mocha",
+            "type": "node",
+            "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
+            "stopOnEntry": false,
+            // Command line arguments passed to the program.
+            "args": [
+                "--ui",
+                "tdd",
+                // "--grep", "androidPlatform",
+                "out/test/debugger/**/*.test.js"
```

Comment:
This won't include the common tests, which are also run in a debugger context as well.